etw/cs/traceloggingdynamic/tests/Program.cs

```diff
@@ -55,7 +97,8 @@ static void Main()
 
                 eb.Reset("structs");
                 eb.AddUInt8("start", 0);
-                eb.AddStruct("struct1", 2, 0xFE00000);
+                var structPos = eb.AddStruct("struct1", 1, 0xFE00000);
+                eb.SetStructFieldCount(structPos, 2);

```

Comment:
This could be added after line 104 also, right?

Does this allow me to start with `AddStruct("struct1", 1)`, then add more than one field and finally call `eb.SetStructFieldCount(structPos, numberOfFieldsAdded)`.

Reply:
Yes, that is what it does.

Reply:
Specifically, you call SetStructFieldCount(structPos, numberOfFieldsInStruct), and it overwrites the value you set in AddStruct.

Reply:
It would be good to have the example show that by explicitly calling `SetStructFieldCount` after adding more fields than what was specified during `AddStruct`.

Reply:
The struct has two fields. We set it to 1 then change it to 2.

Reply:
Yes, but if we actually added the two fields first before calling `SetStructFieldCount`, it would demonstrate the use case well as opposed to just immediately updating it in the very next line. But I guess, this is more of a test code than "example" code.